torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -520,6 +520,7 @@ RegisterOperators reg({
             };
           }
         }),
+    Operator("aten::_unwrap_optional(t? optional) -> t", noop),
```

Comment:
Throw an exception if the IValue is none.

Reply:
In the python-version of the function as well I assume ?

Reply:
yes, I suppose so